JobConfig/digitize/prolog.fcl

```diff
@@ -92,6 +92,23 @@ Digitize: {
     "keep mu2e::CaloClusters_CaloClusterFast_*_*",
     "keep mu2e::ProtonBunchIntensity*_*_*_*"
   ]
+  DTSProducts : [
+    "keep mu2e::StrawGasSteps_compressDetStepMCs_*_*",
```

Comment:
Reformatting this as  "keep \*", drop  "\*\_\*\_\*\_DTSFilter"   will  be more future-proof, in case we add new products.  We want to keep all the inputs and drop anything created by this processing (not sure what that is, I put in DTSFilter)